[13.0][IMP] sale_stock_picking_note: Add Picking comments in sale orders to use in pickings (#1698)
OCA/sale-workflow: sale_stock_picking_note/README.rst

|badge1| |badge2| |badge3| |badge4| |badge5| 

> This module extend sale order to include a picking note that will be set as the picking
> note.
> This module extend sale order to include a picking comments that will be set as the picking
> comment.

Reviewer: ```suggestion
This module extends sales order to include both a external (customer) and internal picking note 
that will be transferred to the picking.
```

#. Place a new sale order with stockable products.

> #. Go to *Other information* tab and set the **Picking Note** you want to.
> #. Go to *Other information* tab and set the **Picking Comments** you want to.

Reviewer: Change labels for being more understandable:

```suggestion
#. Go to *Other information* tab.
#. Set the **Picking Internal Note** and **Picking Customer Comments** you want to.
```

#. Confirm the Sale Order and go to the created picking.

> #. The note is available in the "Note" tab.
> #. The comnments is available in the "Comments" tab.

Reviewer: Put both together in the same tab

```suggestion
#. On the "Comments" tab, you will see both the internal note and customer comments.
```
